test/onnx/test_onnx_export.py

```diff
@@ -19,27 +20,73 @@
 sys.path.append(pytorch_test_dir)
 
 
+def export_to_onnx(
+    model: Union[torch.nn.Module, torch.jit.ScriptFunction],
+    input: Tuple[torch.Tensor],
+    custom_ops: Optional[
+        Iterable[
+            Tuple[
+                str,
+                Union[contextlib.AbstractContextManager, contextlib.ContextDecorator],
+                int,
+            ]
+        ]
+    ] = None,
+    mocks: Optional[Iterable[unittest.mock.patch]] = None,
+    operator_export_type: OperatorExportTypes = OperatorExportTypes.ONNX,
+    opset_version=torch.onnx.symbolic_helper._export_onnx_opset_version,
+) -> onnx.ModelProto:
+    """Exports `model(input)` to ONNX and returns it.
+
+    Custom operators and/or unittest patches can be used help reproducing specific behaviors.
+
+    Args:
+        model: model to export
+        input: model input with same format as `torch.onnx.export(..,args,...)`
+        custom_ops: list of custom operators to use during export
+        mocks: list of mocks to use during export
+        operator_export_type: export type as described by `torch.onnx.export(...operator_export_type,...)`
+        opset_version: ONNX opset version as described by `torch.onnx.export(...opset_version,...)`
+    Returns:
+        A valid ONNX model (`onnx.ModelProto`)
+    """
+    custom_ops = custom_ops or []
+    mocks = mocks or []
+    with contextlib.ExitStack() as stack:
+        for mgr in [*custom_ops, *mocks]:
```

Comment:
Avoid abbreviations. https://google.github.io/styleguide/pyguide.html#316-naming

```suggestion
        for context_manager in [*custom_ops, *mocks]:
```

Reply:
Optionally use `itertools.chain` for `[*custom_ops, *mocks]` to avoid the need for constructing a new list. https://docs.python.org/3/library/itertools.html#itertools.chain